e2e/cypress/integration/api/iam/02_policies_api.spec.ts

```diff
@@ -203,13 +203,14 @@ describeIfIAMV2('policies API', () => {
         });
```

Comment:
Not shown but should change L113 from this
```
      headers: { 'api-token': nonAdminToken },
```
to
```
      headers: {  }, // must fill in before use
```
It looks like it is doing something useful but it is deceptive because at that point `nonAdminToken` is an empty string by definition.